atd-toolbox/data_model/python/debug_record_mismatched/debug.sql

```diff
@@ -0,0 +1,61 @@
+-- all fatal crashes in the old schema that are missing in the new, based purely on the `atd_fatality_count` column
+with newdm as (
+    select
+        id,
+        cris_crash_id,
+        vz_fatality_count
+    from
+        crashes_list_view
+    where
+        vz_fatality_count > 0
+        and private_dr_fl = FALSE
+        and in_austin_full_purpose = TRUE
+),
+
+joined_crashes as (
+    select
+        old.crash_id,
+        newdm.cris_crash_id as new_crash_id
+    from atd_txdot_crashes as old
+    left join newdm on old.crash_id = newdm.cris_crash_id
+    where
+        old.crash_date >= '2014-01-01'
+        and old.crash_date < '2024-01-01'
```

Comment:
yes, thanks for flagging this. until very recently i have excluded 2024 crashes just because it's a moving target. there are new crashes and VZ edits coming in every day.